### PR TITLE
Tagging aspects for Java fundamentals

### PR DESCRIPTION
**Pull request by ryanwarsaw**
- [x] Tag all of Java fundamental insights with aspects.
- [x] Add code tag to ``` blocks where missing.
- [x] Remove excess whitespace, correct some bad insights (interface insight was incorrect).

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/intro-to-java/compiling-and-running-java-code.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/intro-to-java/java-hello-world.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/intro-to-java/java-versions.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/intro-to-java/what-is-java.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/intro-to-java/why-should-i-learn-java.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/java-basics/functions.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/java-basics/java-identifiers.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/java-basics/java-primitives.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/java-basics/java-strings.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/java-basics/packages.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/modifiers/default-and-public-modifiers.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/modifiers/encapsulation.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/modifiers/java-modifiers.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/modifiers/the-final-modifier.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/modifiers/the-private-and-protected-modifiers.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/modifiers/the-static-and-final-modifiers.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/modifiers/the-static-modifier.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/operators-i/constant-notation.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/operators-i/difference-between-jre-and-jdk.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/operators-i/different-types-of-variables-in-java.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/operators-i/ternary-operator.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/operators-i/use-meaningful-names.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/fundamentals/operators-ii/use-varargs-when-unsure-about-the-number-of-parameters.md?ref=babcf6e8b7ef3bdeaa75413f3faeba473a4b8574

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - java/fundamentals/object-oriented-paradigm/java-interfaces.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - java/fundamentals/object-oriented-paradigm/java-interfaces.md

File: java/fundamentals/intro-to-java/what-is-java.md [hunk 1]
```diff
@@ -27,30 +28,13 @@ One of the particularities of the language that makes it so widely used is that
...
-The most recent major Java release is *Java SE 10*, or simply `Java 10`.
-
 In short, Java is trying to be an object-oriented, secure, portable, threaded and dynamic language. We will discuss all these terms in future insights.
 
 *Note*: You should get comfortable with the acronyms as they will come up regularly in the course of the workouts.
 
 ---
 ## Practice
 
-What's the latest version of **Java**?
```
[the remark this answers is not in the document]
Comment (ryanwarsaw) [reply java/fundamentals/intro-to-java/what-is-java.md]: It's not an empty question, it's formatted as such:

```
---
## Practice

What type of programming language is **Java** considered?

???

* object-oriented
* functional
* quantum
* esoteric
```

File: java/fundamentals/iteration/diamond-operator.md [hunk 2]
```diff
@@ -60,7 +63,7 @@ Map<String, List<Integer>> map =
 
 Which of the following Java code snippets is most efficient and illustrates the correct use of a Diamond Operator? 
 
-```
+```java
 //Example A
```
[the remark this answers is not in the document]
Comment (ryanwarsaw) [reply java/fundamentals/iteration/diamond-operator.md]: 👍

**Pull request merged**